Source/WebCore/animation/WebAnimationTypes.h

```diff
@@ -64,6 +67,33 @@ using AnimationEvents = Vector<Ref<AnimationEventBase>>;
 using PropertyToTransitionMap = HashMap<CSSPropertyID, RefPtr<CSSTransition>>;
 using CSSAnimationCollection = ListHashSet<RefPtr<CSSAnimation>>;
 
+using AnimatableProperty = std::variant<CSSPropertyID, AtomString>;
+using AnimatablePropertyToTransitionMap = HashMap<AnimatableProperty, RefPtr<CSSTransition>>;
+
 } // namespace WebCore
 
+namespace WTF {
+
+template<> struct DefaultHash<WebCore::AnimatableProperty> {
```

Comment:
We should set up default hashing for `std::variant` when all values in the variant can be hashed so this doesn’t have to be done specifically for this variant.

Reply:
Yes, that would have made my life easier :)